test/csuite/schema_abort/main.c

```diff
@@ -793,11 +778,13 @@ thread_run(void *arg)
             switch (__wt_random(&td->data_rnd) % 20) {
             case 0:
                 WT_RELEASE_WRITE_WITH_BARRIER(th_ts[td->info].op, BULK);
-                test_bulk(td);
+                if(!use_txn)
```

Comment:
Another idea, if `use_txn` is enabled, ignore it and still proceed with the execution of this function, and make sure we don't do anything related to transaction inside that function. What do you think?